HPC/Gromacs.md

```diff
@@ -0,0 +1,95 @@
+Gromacs is one of the most widely used molecular dynamics software packages. One important way for Gromacs to achieve high performance is to use Modern processors' SIMD (single instruction multiple data) capabilities to increase floating-point performance. Below is the recommended SIMD option for different architectures:
+* Hpc7g/c7gn (-DGMX_SIMD=ARM_SVE)
+* c6gn (-DGMX_SIMD=ARM_NEON_ASIMD)
+We noticed that on the same Hpc7g instance, which supports both SVE and NEON, choose ARM_SVE as SIMD option gives 10% better performance than ARM_NEON_ASIMD. 
+
+## Build Gromacs
+* follow the instructions to install [ArmPL](README.md#libraries) and build Gromacs with ArmPL fft library
+```
+export LDFLAGS="-lgfortran -lamath -lm -lastring"
+cmake .. -DGMX_BUILD_OWN_FFTW=OFF \
+      -DREGRESSIONTEST_DOWNLOAD=ON \
+      -DCMAKE_C_FLAGS="-mcpu=neoverse-512tvb --param=aarch64-autovec-preference=4 -g" \
+      -DCMAKE_CXX_FLAGS="-mcpu=neoverse-512tvb --param=aarch64-autovec-preference=4 -g" \
+      -DCMAKE_C_COMPILER=$(which mpicc) \
+      -DCMAKE_CXX_COMPILER=$(which mpicxx) \
+      -DGMX_OMP=ON \
+      -DGMX_MPI=ON \
+      -DGMX_SIMD=ARM_SVE \
+      -DGMX_BUILD_MDRUN_ONLY=OFF \
+      -DGMX_DOUBLE=OFF \
+      -DCMAKE_INSTALL_PREFIX=${CURDIR} \
+      -DBUILD_SHARED_LIBS=OFF \
+      -DGMX_FFT_LIBRARY=fftw3 \
+      -DFFTWF_LIBRARY=${ARMPL_LIBRARIES}/libarmpl_lp64.so \
+      -DFFTWF_INCLUDE_DIR=${ARMPL_INCLUDES} \
+      \
+      -DGMX_BLAS_USER=${ARMPL_LIBRARIES}/libarmpl_lp64.so \
+      -DGMX_LAPACK_USER=${ARMPL_LIBRARIES}/libarmpl_lp64.so \
+      \
+      -DGMX_GPLUSPLUS_PATH=$(which g++) \
+      -DGMXAPI=OFF \
+      -DGMX_GPU=OFF
+
+```
+* configure Gromacs with FFTW
```

Comment:
In case they don't want to install ArmPL or they want to compare.

Reply:
For specific work-arounds and special settings we should document, but what is the point of documenting build steps that should be in the Gromacs repository for its dependencies already?  The versions and directions here are likely to go stale.

Reply:
In this guide we provided instructions on how to build for SVE (a new feature added in Gromacs 2022.4; previous release 2021.5 only supports NEON) which gives the best performance on Graviton. Also we showed how to use ArmPL FFT library instead of fftw.